cohortextractor2/dsl.py

```diff
@@ -55,7 +55,7 @@
 from __future__ import annotations
 
 from dataclasses import dataclass
-from typing import Union
+from typing import Mapping, Union
```

Comment:
typing.Mapping was deprecated in 3.9, shall we use collections.abc.Mapping directly to avoid future churn?

Reply:
Oh yes, thank you. I've pulled this change out, actually.